src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -475,6 +475,56 @@ public ErrorDiagnostic ArgumentCountMismatch(int argumentCount, int mininumArgum
                 TextSpan,
                 "BCP087",
                 "Array and object literals are not allowed here.");
+
+            public ErrorDiagnostic UnableToFindPathForModule() => new ErrorDiagnostic(
+                TextSpan,
+                "BCP088",
+                "Unable to find file path for module.");

```

Comment:
Can this say what path was specified?

Reply:
And which module?

Reply:
Note - this is a 'missing syntax' error. I'll update the message to clarify a bit. Generally we've been relying on the position rather than including the identifier in each error message; if we want to change this convention, we should probably do this across all error messages.

Reply:
Shouldn't it say "Expected ... at this location" or something like that?